hypothesis-python/src/hypothesis/internal/conjecture/engine.py

```diff
@@ -92,6 +96,10 @@
 Ls: TypeAlias = list["Ls | int"]
 
 
+def shortlex(s):
+    return (len(s), s)
```

Comment:
seems ridiculous that we didn't have this defined anywhere before, but apparently not!

Reply:
Not by this name, but this is really just `sort_key` renamed!